charts/datadog/templates/_container-otel-agent.yaml

```diff
@@ -3,10 +3,10 @@
   image: "{{ include "image-path" (dict "root" .Values "image" .Values.agents.image) }}"
   imagePullPolicy: {{ .Values.agents.image.pullPolicy }}
   {{- if eq .Values.targetSystem "linux" }}
-  command: ["otel-agent", "--config={{ template "datadog.otelconfPath" . }}/otel-config.yaml"]
+  command: ["otel-agent", "--config={{ template "datadog.otelconfPath" . }}/otel-config.yaml", "--core-config={{ template "datadog.confPath" .  }}/datadog.yaml"]
```

Comment:
do you know if the option is compatible with all agent version? 
if not, are we sure the unknown argument will be ignore at start time and not trigger a process crash?

Reply:
Honestly, I don’t think we should have to make any provisions for older versions (given that we’re at a beta stage for the `otel-agent`). I think other than our first beta images from September, this will work well. We should probably document this in the OTel beta page however.